modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/api.ts

```diff
@@ -78,7 +81,7 @@ function getValidFields({
 		fields.push({
 			format: properties[propertyKey].format,
 			label: propertyKey,
-			name: propertyKey,
+			name: `${contextPath}${propertyKey}`,
```

Comment:
`contextPath` here is just a local internal mechanism, to build hierarchical `name`, delimited by a dot.

I considered extending `FDSField` persistence to include `contextPath`, but that approach quickly increased in complexity. We are using `name` as a unique identifier in a lot of places. All those places would need to be changed to check both `name` and `contextPath`. On the other hand, complex names maintain their uniqueness, and I have not found any blocker to using them.